build-dbt-cross-compiler/BUILD-go/aarch64-elf/TEMPLATE-BUILD.go

```diff
@@ -9,6 +9,7 @@ var Toolchain = cc.GccToolchain{
 	Cpp:     in("bin/aarch64-elf-gcc -E"),
 	Cxx:     in("bin/aarch64-elf-g++"),
 	Objcopy: in("bin/aarch64-elf-objcopy"),
+	Objcopy: in("bin/aarch64-elf-ld"),
```

Comment:
That does not seem right? Has it been tested?

Reply:
Thanks for noticing